tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Gateway_Test.php

```
@@ -506,6 +506,66 @@ public function test_payment_fields_outputs_fields() {
+	 * Test that payment_scripts registers the wc-stripe-upe-classic script with the correct version and dependencies.
+	 *
+	 * Because build/upe-classic.asset.php may not be present in test environments, we have conditional logic as follows:
+	 *  - When build/upe-classic.asset.php exists, we verify the script version and dependencies from that file are used.
+	 *  - When build/upe-classic.asset.php does not exist, we verify the fallback values are used.
+	 */
+	public function test_payment_scripts_registers_script_with_correct_version(): void {
+		$asset_path = WC_STRIPE_PLUGIN_PATH . '/build/upe-classic.asset.php';
+
+		// Determine the expected version without modifying any files: if the compiled asset file
+		// is present (i.e. a build has been run), mirror the same logic used in payment_scripts().
+		$expected_version      = WC_STRIPE_VERSION;
```

PHPMD (2.15.0)

[error] The variable $expected_version is not named in camelCase. (undefined)

(CamelCaseVariableName)